src/App.js

```diff
@@ -1,22 +1,47 @@
 import React from "react";
-import "./App.css";
-import ToggleSwitch from "./components/ToggleSwitch";
+import styled from "styled-components";
+import SettingsBox from "./components/SettingsBox";
+import Head from "./components/Head";
+
+// https://coolors.co/282c34-4fbe79-39a2ae-434f4f-7a9e9f
+
+const Playground = styled.div`
+  height: 100vh;
+  width: 100vw;
+  background-color: #282c34;
+`;
+
+const TheGrid = styled.div`
+  height: fit-content;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+  grid-template-rows: repeat(auto-fit, minmax(120px, 1fr));
+`;
 
 function App() {
   const [settings, setSettings] = React.useState({
-    darkmode: false
+    darkmode: false,
+    sound: true,
+    skull: false,
+    settings: true
   });
 
-  function handleToggleDarkmode(value) {
-    setSettings({
-      darkmode: value
-    });
+  console.log(`The current settings are:`, settings);
+
+  function saveSettings(newSettings) {
+    setSettings(newSettings);
   }
 
   return (
-    <div className="App">
-      <ToggleSwitch settings={settings} toggleDarkmode={handleToggleDarkmode} />
-    </div>
+    <Playground>
+      <TheGrid>
+        {settings.settings && (
+          <SettingsBox settings={settings} saveSettings={saveSettings} />
```

Comment:
SettingsBox is controlled and have a value and change event. 
You called the value `settings` which is fine.
But the change event is called `saveSettings`, which is not a good name for an event.
I would name it `onSettingsChange`.

Or you use `value` and `onChange`  like in common components.